### DISCUSSION TITLE
Static code checks when working separately

### DISCUSSION THREAD

**Discussion by etonotieno** [droidconKE/droidconKE2023Android#222] [General]
When `codeAnalysis` is run, `ktlintFormat` & `spotlessApply` may apply formatting to files outside your work scope. From the contributing guidelines, you should only make changes related to your work. Should these changed files be included as part of your PR?
@chepsi @wangerekaharun @michaelbukachi @jumaallan @kibettheophilus @misshannah @MamboBryan

**Comment by kibettheophilus** [comment]
@etonotieno for code formatting, IMO I don't think that should be an issue.
